utils/mock-data-loader/database/user-repository.js

```diff
@@ -23,14 +23,19 @@ const issueValid2faJWT = (user, sessionIdentifier) => {
 
 const createLoggedInUserSession = async (user) => {
   try {
+    console.info('Hello 123123');
+    console.info(process.env.MONGODB_URI);
     const userCollection = await db.getCollection('users');
+    console.info(userCollection);
+    console.info('7784778478478478');
     const userFromDatabase = await userCollection.findOne({ username: { $eq: user.username } }, { collation: { locale: 'en', strength: 2 } });
 
     const sessionIdentifier = crypto.randomBytes(32).toString('hex');
     const token = issueValid2faJWT(userFromDatabase, sessionIdentifier);
     await userCollection.updateOne({ _id: { $eq: userFromDatabase._id } }, { $set: { sessionIdentifier } });
     return `Bearer ${token}`;
   } catch (e) {
+    console.error('Failed to create %O', e);
```

Comment:
We're using `%o` elsewhere after Abhi's ticket is merged
```suggestion
    console.error('Failed to create %o', e);
```
fwiw as I had to look it up again:

If specifier is %o, optionally let converted be current with optimally useful formatting applied.

If specifier is %O, optionally let converted be current with generic JavaScript object formatting applied.